src/main/java/nl/novi/techiteasycontroller/Controller/TelevisionController.java

```diff
@@ -0,0 +1,64 @@
+package nl.novi.techiteasycontroller.Controller;
+
+import nl.novi.techiteasycontroller.Dto.Output.TelevisionOutputDto;
+import nl.novi.techiteasycontroller.Exceptions.RecordNotFoundException;
+import nl.novi.techiteasycontroller.Model.Television;
+import nl.novi.techiteasycontroller.Services.TelevisionService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Optional;
+
+@RequestMapping("television")
+@RestController
+public class TelevisionController {
+
+    private final TelevisionService televisionService;
+
+    public TelevisionController(TelevisionService televisionService) {
+        this.televisionService = televisionService;
+    }
+
+
+    @GetMapping()
+    public ResponseEntity<List<TelevisionOutputDto>> getAllTelevisions() {
+        List<TelevisionOutputDto> Televisionoutput = televisionService.getAllTelevisions();
+        return ResponseEntity.ok(Televisionoutput);
+    }
+
+    @PostMapping
+    public ResponseEntity<Object> createTelevision(@RequestBody TelevisionOutputDto tdto, BindingResult br) {
+
+        if (br.hasFieldErrors()) {
+            StringBuilder sb = new StringBuilder();
+            for (FieldError fe : br.getFieldErrors()) {
+                sb.append(fe.getField() + ": ");
+                sb.append(fe.getDefaultMessage() + "\n");
+            }
+            return new ResponseEntity<>(sb.toString(), HttpStatus.BAD_REQUEST);
+        }
+
+        Long id = televisionService.createTelevision(tdto);
+        tdto.id = id;
```

Comment:
Jou service methode returned een ID, dat is prima. Vervolgens voeg je dat id toe aan de inputDto (hier een outputDto, maar het is bedoeld als inputDto) en return je die dto (met aangepast ID) in de ResponseEntity. Als je een DTO in de responseEntity wilt returnen, kun je het best ook een DTO uit de service returnen en die doorgeven in de ResponseEntity. Hoe je het nu doet loop je namelijk het risico dat er in de service een aanpassing gedaan wordt aan de data in de dto en dat je dat niet weergeeft in de dto die je returned naar de gebruiker (want de enige aanpassing die je doorvoert is immers het ID). Lang verhaal, maar ik hoop dat je begrijpt wat ik bedoel